File: app/models/recipe.rb
class Recipe < ApplicationRecord
  validates :name, presence: true
  validates :description, presence: false
  validates :minutes, presence: true
  belongs_to :user
  has_many :instructions
  has_many :ingredients

  MINUTES = [0, 5,10, 15,
            20, 25, 30, 35,
            40,45, 50, 55]

  HOURS = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
end

File: react/src/components/recipes/Recipe.js
import React from 'react'

const Recipe = (props) => {
  return (
    <div className="recipe-item">
      <h4><a href={`http://localhost:3000/recipes/` + props.id}>{props.name}</a></h4>
        <p>{props.description}</p>
        <p>{props.ingredients}</p>
        <div className="time">
          <p>{props.hours} hr | {props.minutes} min </p>
        </div>
    </div>
  )
}

export default Recipe;
